prac_05/word_occurrences.py

```diff
@@ -0,0 +1,19 @@
+"""
+Word Occurrences Prac 05
+Estimated time:15 minutes
+Actual:21 minutes
+"""
+
+
+word_count = {}
+text = input("Text: ")
+strings = text.split()
+for word in strings:
+    occurrences = word_count.get(word, 0)
+    word_count[word] = occurrences + 1
+strings = list(word_count.keys())
+strings.sort()
```

Comment:
the variable "string" is used to store words, use more descriptive name such as "words"